### PR TITLE
adding entity counter recalculator

### PR DESCRIPTION
**Pull request by JackLewis-digirati**
This pull request adds a python lambda function that recalculates the entity counter table to be accurate, based on the images table

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - name: Install dependencies
+        run: python -m pip install -r requirements.txt
+      - name: Run python tests
+        working-directory: ./entity-counter-recalculator
+        run: python -m unittest test_main.py
+      - name: Install zip
+        uses: montudor/action-zip@v1
+      - name: Zip output
+        working-directory: ./entity-counter-recalculator
+        run: zip -qq -r entity-counter-recalculator_${{ github.run_id }}.zip main.py app/
```
Comment (donaldgray) [inline .github/workflows/deploy.yaml]: The uploaded zip doesn't look like it contains any of the dependencies from `requirements.txt`, it's just the .py files?

File: entity-counter-recalculator/event.json [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (donaldgray) [inline entity-counter-recalculator/event.json]: Is there a sample payload that could be included here?

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+moto==4.1.15
+packaging==23.1
+paramiko==3.3.1
+pluggy==1.2.0
+psycopg2==2.9.7
+pycparser==2.21
+PyNaCl==1.5.0
+pytest==7.4.0
+python-dateutil==2.8.2
+python-lambda-local==0.1.13
```
Comment (donaldgray) [inline requirements.txt]: Is this a big dependency? If it bloats the resulting zip file is it worth having 2 separate requirements? 1 that has the key deps and another for local-dev specific stuff?

File: entity-counter-recalculator/event.json (hunk 2 above)
Comment (JackLewis-digirati) [reply entity-counter-recalculator/event.json]: This is actually the event.json required to run the lambda using python-lambda-local.  However, given we're moving towards using a docker image, I can probably remove this along with the code related to running a lambda in python directly

File: requirements.txt (hunk 3 above)
Comment (JackLewis-digirati) [reply requirements.txt]: I've now moved this to create a `minimalRequirements.txt` file that will remove packages required for testing

**Pull request closed without merging**